internal/docstore/mongodocstore/mongo.go

```diff
@@ -121,7 +121,7 @@ func (c *collection) create(ctx context.Context, a *driver.Action) error {
 	if err != nil {
 		return err
 	}
-	mdoc[docstore.RevisionField] = int64(1)
+	mdoc[docstore.RevisionField] = 1
```

Comment:
I suggest to use the unique string as the revision. This field is an implementation detail and used only by the library, not read by human. Also I think use may implement some human-readable version if they have their specific needs so we want to avoid use a version-like revision here the prevent confuse them.

Reply:
Done.